exec/java-exec/src/main/java/org/apache/drill/exec/ops/QueryContext.java

```diff
@@ -61,6 +61,7 @@
 // in fragment contexts
 public class QueryContext implements AutoCloseable, OptimizerRulesContext, SchemaConfigInfoProvider {
   private static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(QueryContext.class);
+  public enum StatementType {UNKNOWN, ANALYZE, CTAS, EXPLAIN, REFRESH, SELECT, SETOPTION};
```

Comment:
There are several other SQL statement types which require additional code to figure out (Please see child classes of DefaultSqlHandler). I have renamed the UNKNOWN type to OTHER. For now, this is sufficient since there is no custom logic for these statement types e.g. VIEWs/DESCRIBEs/DROPs. If need be, we can extend the statement types and add the logic to determine if the new statement type.